packages/react-accordion/src/Accordion.stories.tsx

```diff
@@ -71,10 +64,10 @@ AccordionExample.argTypes = {
     defaultValue: false,
     control: 'boolean',
   },
-  circular: {
-    defaultValue: false,
-    control: 'boolean',
-  },
+  // circular: {
+  //   defaultValue: false,
+  //   control: 'boolean',
+  // },
```

Comment:
Why this is commented?

Reply:
Still not supported by tabster. I'll delete it!